src/park_management_gui.cpp

```diff
@@ -41,6 +41,7 @@ class ParkManagementGui : public GuiWindow {
  * @ingroup gui_group
  */
 enum ParkManagementWidgets {
+	/* The tabbutton constants must be in sync with #ParkManagementGuiTabs. */
```

Comment:
If you add such comments in the doxygen text above, it becomes visible in the documentation as well. Also, stuff like "#ParkManagementGuiTabs" make more sense then.

Not sure if you did that, but add such a note in ParkManagementGuiTabs as well pointing to here.

Of course if you really want to be sure it should be enforced or checked, but that may be a bit too much here.